src/sass/components/_index.scss

```
@@ -4,3 +4,7 @@
 @import './backdrop';
 @import './open-menu-btn';
 @import './navigation';
+@import './modal-window';
+@import './callback-form';
+@import './form-components/custom-checkbox';
+@import './form-components/input-elemet';
```

```suggestion
@import './form-components/input-element';
```
если будешь править, то и в имени файла тоже